src/env/env.cpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by pmerk on 22/03/2021.
+//
+
+#include "env/env.hpp"
+#include <cstdlib>
+
+using namespace CGIenv;
+
+env::env() : _envp(NULL) { }
+
+env::~env() {
+	if (_envp) {
+		for (int i = 0; _envp[i]; i++)
+			free(_envp[i]);
+		free(_envp);
```

Comment:
free using delete (see other review comments)

Reply:
changed in latest commit